synapse/lib/cell.py

```diff
@@ -310,6 +310,15 @@ def _getAuthItem(self, name):
 
         raise s_exc.NoSuchName(name=name)
 
+    async def getCellCmdTypes(self):
```

Comment:
Lets change the terminology here to cell "interfaces" so it's less specific to commands and also give those interfaces versions so we can do a bit of min-version backward compat stuff if we want.